projects/ansible/build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -eu
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+
+# Change config = ConfigManager()
+# to
+# config = ConfigManager("/tmp/base.yml")
+# 
+# The init function of ConfigManager is executed when
+# a package that uses the ConfigManager is imported.
+# We therefore create /tmp/base.yml at fuzz-time.
+sed 's/config = ConfigManager()/config = ConfigManager(defs_file="\/tmp\/base.yml")/g' -i /src/ansible/lib/ansible/constants.py
+
+# Change 'with open(to_bytes(metadata_path))'
+# to
+# 'with open("/tmp/ansible_builtint_runtime.yml")'
+sed 's/to_bytes(metadata_path)/"\/tmp\/ansible_builtin_runtime.yml"/g' -i /src/ansible/lib/ansible/utils/collection_loader/_collection_finder.py
+
+# Change 'args_parser.parse()'
+# to
+# args_parser.parse(skip_action_validation=True)
+sed 's/args_parser.parse()/args_parser.parse(skip_action_validation=True)/g' -i /src/ansible/lib/ansible/playbook/task.py
```

Comment:
Why is action validation skipped?

Reply:
The reason for this is that if we allow the fuzzer to go into the `else` here:
https://github.com/ansible/ansible/blob/f8ef34672b961a95ec7282643679492862c688ec/lib/ansible/parsing/mod_args.py#L303-L309

... then we run into this error:

```
=== Uncaught Python exception: ===
ModuleNotFoundError: No module named 'ansible.plugins.action'
Traceback (most recent call last):
  File "load_list_of_tasks_fuzzer.py", line 53, in TestOneInput
  File "ansible/playbook/task.py", line 152, in load
  File "ansible/playbook/base.py", line 222, in load_data
  File "ansible/playbook/task.py", line 220, in preprocess_data
  File "ansible/parsing/mod_args.py", line 309, in parse
  File "ansible/plugins/loader.py", line 538, in find_plugin_with_context
  File "ansible/plugins/loader.py", line 629, in _resolve_plugin_step
  File "ansible/plugins/loader.py", line 659, in _find_plugin_legacy
  File "ansible/plugins/loader.py", line 344, in _get_paths_with_context
  File "ansible/plugins/loader.py", line 306, in _get_package_paths
```

I am sure this can be circumvented, with additional configuration, but for now I allowed the fuzzer to go a bit deeper by skipping action validation. I will be happy to improve upon this fuzzer progressively look into the missing plugin.